### PR TITLE
Express controller subgenerator and test added

### PR DESCRIPTION
**Pull request by UndeadBaneGitHub**

### CODE REVIEW

File: express-controller/templates/_.server.controller.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+exports.delete = function (req, res) {
+
+};
+
+/**
+ * List of <%= humanizedPluralName %>
+ */
+exports.list = function (req, res) {
+
+};
```
Comment (codydaig) [inline express-controller/templates/_.server.controller.js]: New Line

**Comment by codydaig** [comment]
Left a line comment.
Don't forget to squash.
Otherwise, LGTM.

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "bluebird": "~3.0.6",
     "chalk": "~1.1.1",
+    "underscore.inflections": "^0.2.1",
```
Comment (ilanbiala) [inline package.json]: ~

**Comment by codydaig** [comment]
@ilanbiala LGTY?

Also, what's the status on the other ones? Are there still changes you want made or can you give your approval? I'd like to get these merged in and published.

**Comment by ilanbiala** [comment]
LGTM, timeout PR next.

**Comment by ilanbiala** [comment]
LGTM.

**Comment by codydaig** [comment]
@ilanbiala No worries, It happens.  Just wanted to get these pushed through.

**Pull request merged**